app/notification.go

```diff
@@ -203,6 +204,7 @@ func (a *App) SendNotifications(post *model.Post, team *model.Team, channel *mod
 					}
 
 					if membership != nil && !membership.Following {
+						participantMemberships[userID] = membership
```

Comment:
This is a data race - we are writing to this map from multiple goroutines. Please guard this with a mutex.

Reply:
Right, will do